internal/pkg/cli/deploy/env.go

```diff
@@ -398,6 +410,7 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*deploy.Creat
 		ArtifactBucketARN:    s3.FormatARN(partition.ID(), resources.S3Bucket),
 		ArtifactBucketKeyARN: resources.KMSKeyARN,
 		CIDRPrefixListIDs:    cidrPrefixListIDs,
+		PublicALBSourceIPs:   publicALBSourceIPs,
```

Comment:
nit: delete the variable 
```suggestion
		PublicALBSourceIPs:   d.publicALBSourceIPs(in),
```
https://a-nickels-worth.blogspot.com/2016/04/a-guide-to-naming-variables.html#orgheadline13